README.md

```diff
@@ -141,7 +141,7 @@ and for android run,
 
 Note, you might experience problems that seem to be related to the tests starting the Appium server, for example errors that say `Connection Refused`, `Connection Reset` or `The requested environment is not available`. Sorry about that this is still a WIP, you can manually start the Appium server via [appium desktop](https://github.com/appium/appium-desktop) or the cli, then change the port number in the tests while optionally commenting out related code in the `beforeAll` and `afterAll` block. 
 
-For a more detailed outline of the UI tests and how to get started writing one please visit the [Project Wiki](https://github.com/wordpress-mobile/gutenberg-mobile/wiki/Getting-started-with-UI-tests)
+For a more detailed outline of the UI tests and how to get started writing one please visit the [UI Test documentation](https://github.com/wordpress-mobile/gutenberg-mobile/blob/__device-tests__/README.md)
```

Comment:
What would you think about adding a second link here to the CONTRIBUTING doc so that it is clear that there are two docs? Maybe something loosely along these lines could work:

>For a more detailed outline of the UI tests, including information about how to run them, please visit the [UI Test README](https://github.com/wordpress-mobile/gutenberg-mobile/blob/__device-tests__/README.md). For information about writing your own UI tests, please visit the [UI Test Contributing guide](https://github.com/wordpress-mobile/gutenberg-mobile/blob/__device-tests__/CONTRIBUTING.md).

Reply:
Done